packages/zoe/src/zoeService/zoe.js

```diff
@@ -40,11 +39,25 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
   const instanceToInstanceAdmin = makeWeakStore('instance');
 
   /** @type {GetAmountMath<any>} */
-  const getAmountMath = brand => issuerTable.get(brand).amountMath;
+  const getAmountMath = brand => issuerTable.getByBrand(brand).amountMath;
 
   /** @type {WeakStore<Brand, ERef<Purse>>} */
   const brandToPurse = makeWeakStore('brand');
 
+  /**
+   * Create an opaque handle object.
+   *
+   * @template {string} H
+   * @param {H} handleType the string literal type of the handle
+   * @returns {Handle<H>}
+   */
+  const makeHandle = handleType => {
+    // This assert ensures that handleType is referenced.
+    assert.typeof(handleType, 'string', 'handleType must be a string');
+    // Return the intersection type (really just an empty object).
+    return /** @type {Handle<H>} */ (harden({}));
+  };
```

Comment:
Does this function do anything useful?

Reply:
> To answer #1494, makeHandle is purely to make the typing work. No other reason.

Sorry I missed this. Resolving. (But not happy that typing forces us to add such a silly piece of code.) What is the typing problem this solves?

Reply:
> What is the typing problem this solves?

It was used to make an `InstanceHandle` and an `InstallationHandle` and any others disjoint types, so we caught errors when one was used in the wrong context.

Reply:
Do we still have such handles? Why can't we just use object types?